documentation/source/newsfragments/1798.feature.rst

```diff
@@ -0,0 +1,3 @@
+Support for building with Microsoft Visual C++ has been added.
+The official CPython distribution for Windows is now supported in addition to the mingw version.
```

Comment:
The issue with setuptools_scm means the mingw environments are not currently supported. That might be fixed upstream, or perhaps installing from sdists where setuptools_scm has already run would work? We don't currently have any instructions for how to use the official distribution for Python on Windows. I think we should remove this line.